packages/backend-plugin-api/report.api.md

```diff
@@ -24,6 +24,40 @@ import { Readable } from 'stream';
 import type { Request as Request_2 } from 'express';
 import type { Response as Response_2 } from 'express';
 
+// @public (undocumented)
+export type AuditorCreateEvent<TRootMeta extends JsonObject> = (options: {
+  eventId: string;
+  severityLevel?: AuditorEventSeverityLevel;
+  request?: Request_2<any, any, any, any, any>;
+  meta?: TRootMeta;
+  suppressInitialEvent?: boolean;
+}) => Promise<{
+  success<TMeta extends JsonObject>(options?: { meta?: TMeta }): Promise<void>;
+  fail<TMeta extends JsonObject, TError extends Error>(
+    options: {
+      meta?: TMeta;
+    } & (
+      | {
+          error: TError;
+        }
+      | {
+          errors: TError[];
```

Comment:
What do you think about just `error: TError`? forcing the caller to use something like `AggregateError` or something similar if there are multiple errors. Thinking that keeps things quite a bit simpler.